scrapy/pipelines/files.py

```diff
@@ -284,7 +330,6 @@ def __init__(self, store_uri, download_func=None, settings=None):
 
         if isinstance(settings, dict) or settings is None:
             settings = Settings(settings)
-
```

Comment:
:lipstick:  Please, undo this and the next line removals.

Reply:
When did I do that 😆